services/masterdata/src/lib/core/utils/key-value-pair.class.ts

```diff
@@ -1,3 +1,3 @@
 export class KeyValuePair<TKey, TValue> {
```

Comment:
`KeyValuePair` is already defined in `@shared/core`. Maybe it's better to promote this one to the framework instead of duplicating code.

Reply:
i can do that. just give me the full path and filename to where to put this KeyValuePair?